.travis.yml

```diff
@@ -46,7 +46,10 @@ before_script:
 - chmod +x ./cc-test-reporter
 - "./cc-test-reporter before-build"
 script:
-- mvn clean install -q
+- mvn clean package -DskipTests
+- mvn test -pl eventlogger,common,api,worker,bfd,filter,audit,hpms,mock-hpms
+- export PATIENT_COVERAGE_YEAR=3
+- mvn test -pl e2e-test
```

Comment:
This is meant as temporary, correct?

If so, needs a TODO to revert.

Reply:
Maybe not once we switch from Travis becomes irrelevant